lib/tml/cache_adapters/redis.rb

```diff
@@ -114,7 +114,7 @@ def exist?(key, opts = {})
     data = @cache.exist(versioned_key(key, opts))
     not data.nil?
   rescue Exception => ex
-    warn("Failed to check if key exists: #{ex.message}")
+    warn("Failed to check if key exist: #{ex.message}")
```

Comment:
Similar to the comment changes, I think we are okay keeping the original wording of "exists" instead of "exist"